readthedocs/vcs_support/backends/git.py

```diff
@@ -95,57 +87,52 @@ def get_remote_fetch_refspec(self):
 
         :return: A refspec valid for fetch operation
         """
-
-        if not self.version_type:
-            log.warning(
-                "Trying to resolve a remote reference without setting version_type is not possible",
-                project_slug=self.project.slug,
-            )
-            return None
-
-        # Branches have the branch identifier set by the caller who instantiated the
-        # Git backend.
-        # If version_identifier is empty, then the fetch operation cannot know what to fetch
-        # and will fetch everything, in order to build what might be defined elsewhere
-        # as the "default branch". This can be the case for an initial build started BEFORE
-        # a webhook or sync versions task has concluded what the default branch is.
```

Comment:
This comment seems to be still useful in the new code for the case `if not branch`, right?